content/docs/06-integrations/01-maintenance-policy.md

```diff
@@ -133,26 +133,33 @@ We provide add-on pack updates as follows:
 
-We deprecate and remove packs when a more stable version of the pack is available or when the underlying technology becomes obsolete. We deprecate a pack six months before deleting it. Post deprecation, you will still be able to create new cluster profiles using the pack and deploy clusters that use profiles containing the pack. Our Release Notes announce when a pack is deprecated and a replacement pack is available.
+![Screenshot showing how Palette indicates a pack's stage of deprecation.](/integrations_deprecation-stage.png)
 
+An information icon in the profile stack also displays a message that instructs about required pack versions.
+
+![Screenshot showing message in profile stack that tells you the required pack version to use.](/integrations_deprecation-profile-stack-msg.png)
+
+We adhere to the following stages of deprecation: 
 
 <br />
 
-<!-- - **Deprecated**: When a pack is deprecated, Palette displays a warning. You will still be able to create new cluster profiles using the pack and launch clusters using existing profiles that contain the pack.
+- **Deprecated**: When a pack or a pack version is deprecated, this indicates it will be removed in the future. You will still be able to create new cluster profiles using the pack and launch clusters using existing profiles that contain the pack.
+
+   The pack remains in *Deprecated* state for three months before it moves to *Disabled* state.
 
-  The pack remains in *Deprecated* state for three months before it moves to *Disabled* state.
+   <br />
 
-- **Disabled**: When a pack is disabled, you will no longer be allowed to create new cluster profiles using the pack. Palette displays a message informing you to use a newer version of the pack. However, you can still deploy new clusters using existing profiles that contain the pack.
+- **Disabled**: When a pack is disabled, it is no longer available for selection in Palette. When creating new profiles, you must use a newer version of the pack.
 
   The pack remains in *Disabled* state for three months before it is deleted.
 
-- **Deleted**: When a pack is deleted, you will not be able to create new cluster profiles that use it or launch new clusters with existing profiles that contain the pack. Palette displays a message informing you to use a newer version of the pack. Existing clusters are not disturbed and will continue to be functional.
+  <br />
 
-- **Deleted**: When a pack is deleted, you will not be able to create new cluster profiles that use it or launch new clusters with existing profiles that contain the pack. You must use a newer version of the pack. Existing clusters that use the pack are not disturbed and will continue to be functional. 2 -->
+- **Deleted**: When a pack is deleted, it is removed from Palette. However, existing clusters that use profiles which contain the deleted pack will continue to function. 
```

Comment:
@ritawatson I think we need to clarify here that an active cluster containing the deleted pack will continue to operate. However, if they try to deploy the cluster profile that contains the missing pack, they will be unable to do so.

Reply:
Revised as follows: 

When a pack is deleted, it is removed from Palette. An active cluster that contains the deleted pack will continue to operate. However, you will not be able to deploy a new cluster profile that contains the deleted pack.